e2e/suite_test.go

```diff
@@ -0,0 +1,113 @@
+// These tests use Ginkgo (BDD-style Go testing framework). Refer to
+// http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
+
+package e2e
```

Comment:
Why do we need another copy of controller `suite_test.go`?

Reply:
Though the setup is similar, but we do not want mix them. 
Also, I will add more e2e tests in the coming PR(s)